.gitignore

```diff
@@ -1,7 +1,6 @@
 
 ###> symfony/framework-bundle ###
 /.env
-/public/bundles/
```

Comment:
@daria-sieroshtan 
please revert this change
Meanwhile you need other PR with necessary assets